.changeset/gold-birds-happen.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/plugin-org': patch
+---
+
+For the component `EntityMembersListCard` you can now specify the type of members you have in a group. For example:
+
+```tsx
+<Grid item xs={12}>
+  <EntityMembersListCard memberType="Ninja's" />
```

Comment:
given the change, think the example in the changeset needs an update?  `memberDisplayTitle`?

Reply:
Nice catch!